torch/csrc/jit/codegen/cuda/scheduler.cpp

```diff
@@ -185,15 +187,19 @@ bool scheduleFusion(Fusion* fusion, const at::ArrayRef<c10::IValue> inputs) {
         continue;
       TensorView* tv = val->as<TensorView>();
 
-      // Should be true for all intermediates, but if one isn't hooked
-      // up right, skip it and hope for the best for now
-      if (!disable_unroll && tv->nDims() == 3) {
-        tv->axis(-2)->parallelize(ParallelType::Unroll);
-        tv->axis(-1)->parallelize(ParallelType::TIDx);
-      } else {
-        if (tv->nDims() == 2)
-          tv->axis(-1)->parallelize(ParallelType::TIDx);
-      }
+      // Disabling below as currently unrolling doesn't make a lot of sense as
+      // we don't extract global loads/reads out of intermediate logic.
+      //
+      // Below check should be true for all intermediates, but if one isn't
+      // hooked up right, skip it and hope for the best for now
+      //
+      // if (!disable_unroll && tv->nDims() == 3) {
+      //   tv->axis(-2)->parallelize(ParallelType::Unroll);
+      //   tv->axis(-1)->parallelize(ParallelType::TIDx);
+      // } else {
+      //   if (tv->nDims() == 2)
+      //     tv->axis(-1)->parallelize(ParallelType::TIDx);
+      // }
     }
     TensorView* out0 = fusion->outputs()[0]->as<TensorView>();
```

Comment:
This is not your problem, but the code below (`out0` & computing `numel`) is obsolete and we can just delete them all as well as the commented out unrolling code.